### PR TITLE
Data update pipeline

### PR DESCRIPTION
**Pull request by allthesignals**
This PR adds a `refresh` command:

`yarn run refresh`

This command copies data from EDM's database schema and loads it into a separate database.

This PR also removes a migration that had never been run on production anyway. The problem I had with the migration was that it was actually not static — it references variables in the application. The problem I saw with that is should those constants be updated, they're effectively changing the actual migration itself. This seemed like bad form.

Instead, our migrations set up the "gateways" to EDM's data through Foreign Data Wrappers but have no opinion about the actual itself...

### CODE REVIEW

File: bin/refresh [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      ${PREV_YEAR}
+      ${DECENNIAL_CUR_YEAR}
+      ${DECENNIAL_PREV_YEAR}
+    Using these remote schema:
+      ${FOREIGN_SCHEMA_NAMES.join(', ')}
+  `);
+
+  doRefresh(); 
+} catch (e) {
+  console.log('Something went wrong, rolling back:', e);
```
Comment (bfreeds) [inline bin/refresh]: @allthesignals thank you for this nice error handling.  I don't fully understand how/where the rollback happens, is this just how the try...catch block works, or a feature of pg-promise to rollback automatically?

File: migrations/1576772885916_add-foreign-data-wrapper.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-const SCHEMAS = ['pff_demographic', 'pff_economic', 'pff_housing', 'pff_social'];
```
Comment (bfreeds) [inline migrations/1576772885916_add-foreign-data-wrapper.js]: Schemas logic removed from here and brought into `refresh` command

**Comment by bfreeds** [comment]
@allthesignals This change - making the migrations run the same every time, regardless of the state of the foreign database, makes sense to me.  If I'm understanding correctly, the current migrations establish the FDW relationship to EDM's database, then the `refresh` command is what accounts for the actual sync of the data.   So the process of updating data would be just `yarn run refresh`.  

I think we should update the `README` so that in the future anyone would be able to follow this process and do the update - let me know if you want to look at this together (I can bring some fresh, ignorant eyes and ask questions to help write-up a beginner-friendly readme).

File: bin/refresh (hunk 1 above)
Comment (allthesignals) [reply bin/refresh]: pg-promise issues the rollback implicitly, so it's the latter: it's a feature of pg-promise. it issues the rollback if any of the promises in `doRefresh` reject.

**Pull request merged**